deploy/traction/values-development.yaml

```diff
@@ -7,6 +7,25 @@ acapy:
       generated: true
     pluginInnkeeper:
       generated: true
+  ledgers.yml:
+    - id: bcovrin-test
+      is_production: true
+      is_write: true
+      genesis_url: "http://test.bcovrin.vonx.io/genesis"
+      endorser_did: "Ket75eV5UQvVkW2XBjgDH7"
+      endorser_alias: "bcovrin-test-endorser"
+    - id: bcovrin-dev
+      is_production: true
+      is_write: true
+      genesis_url: "http://dev.bcovrin.vonx.io/genesis"
+      endorser_did: "VGA8QzjQaL5xkXRHjAgQcj"
+      endorser_alias: "bcovrin-dev-endorser"
+    - id: sovrin-testnet
+      is_production: true
+      is_write: true
+      genesis_url: "https://raw.githubusercontent.com/sovrin-foundation/sovrin/stable/sovrin/pool_transactions_sandbox_genesis"
+      endorser_did: "EZpKx6nT56Hv83JmNz7ik8"
+      endorser_alias: "sovrin-testnet-endorser"
```

Comment:
Was previously using the default values setup here in `values.yaml` so defining the whole thing for the env now.